src/blob/index/core.rs

```diff
@@ -64,9 +64,18 @@ where
     for<'a> K: Key<'a>,
 {
     const KEY_SIZE: usize = K::LEN as usize;
-    const BPTREE_ENTRY_SIZE: usize =
+    const BTREE_ENTRY_SIZE: usize =
         size_of::<Vec<u8>>() + MemoryAttrs::<K>::KEY_SIZE + size_of::<Vec<RecordHeader>>();
     const RECORD_HEADER_SIZE: usize = size_of::<RecordHeader>() + MemoryAttrs::<K>::KEY_SIZE;
+    // Each node in BTreeMap contains preallocated vectors of 11 values and 12 edges.
+    // Although count of nodes can't be determined without reimplementing insertion algorithm,
+    // we can use approximation of overhead size added per one key
+    const BTREE_SIZE_MULTIPLIER: usize = (
+        size_of::<std::ptr::NonNull<()>>() +      // ptr to parent btree node
```

Comment:
According to the source code it is `Option<NonNull<...>>`. `Option` can add 1 byte on top. 
https://doc.rust-lang.org/src/alloc/collections/btree/node.rs.html#52